twine/package.py

```diff
@@ -18,39 +18,35 @@
 import os
 import re
 import subprocess
-import sys
-import warnings
-from typing import (
-    Any,
-    Dict,
-    Iterable,
-    List,
-    NamedTuple,
-    Optional,
-    Sequence,
-    Tuple,
-    Union,
-    cast,
-)
-
-if sys.version_info >= (3, 10):
-    import importlib.metadata as importlib_metadata
-else:
-    import importlib_metadata
-
-import packaging.version
-import pkginfo
+from typing import Any, Dict, List, NamedTuple, Optional, Tuple
```

Comment:
As much as possible (where you're modifying something like this) please split these into separate lines. That avoids merge conflicts. You already did it L23-L24, just be consistent on the better way of formatting

Reply:
Do you mean one `import` statement per symbol imported (as in L23-24) or with parenthesized list of of symbols as in the lines replaced here?

Reply:
Either is fine, but the former is what I was advocating for.

It's also already the pattern below with `from twine import _` and `from packaging import _`.

Reply:
It is a pattern never used in the codebase when importing from `typing`. I wrote it like this for consistency, but if you prefer one import per line, I can change it.

Reply:
The style I used (all symbols on one line) is actually enforced by the configuration of `isort` used for the project. What should I do?